test/plots/population-by-latitude.js

```diff
@@ -0,0 +1,37 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+import {feature} from "topojson-client";
+
+export default async function () {
+  const world = await d3.json("data/countries-50m.json");
+  const land = feature(world, world.objects.land);
+
+  const cities = await d3.csv("data/cities10000.csv", d3.autoType);
+  return Plot.plot({
+    style: {overflow: "visible"},
+    projection: {type: "equirectangular", rotate: [-10, 0]},
+    r: {zero: true, range: [0.5, 6.5]},
```

Comment:
FYI, the zero: true here is being ignored because you’re setting a range explicitly. (I’m tweaking these examples so I’ll fix.)

Reply:
oops too much iteration